eclipse-theia/theia
# vscode: support icons contribution point

#### What it does

Fixes: #12355 

This PR adds support for the `contributes.icon` contribution point from VSCode. This fixes a number of issues where extension icons were invisible such as many of the Gitlens icons.

#### How to test

You can test the functionality of this PR by downloading an extension which contributes its icons through the icons contribution point. One such example is Gitlens.

1.  Open Theia application.
2.  Go to the `Extensions` view
3.  Search and download Gitlens from the vsx-registry.
4.  Verify that all contributed icons are visible and correct.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
